src/Control/Monad/Cofree/Trans.purs

```diff
@@ -0,0 +1,156 @@
+-- | This module defines a lazy implementation of the _cofree monad transformer_.
+
+-- | Given a `CofreeT` f m a:
+-- | - 'f' is a `Functor`, generally representing an AST,
+-- | - 'm' is a 'Monad', generally representing an effect,
+-- | - and 'a' is the type of the annotation.
+-- |
+-- | Usually, you would use `CofreeT` to annotate an existing AST with
+-- | metadata such as source locations, file names, etc.
+
+module Control.Monad.Cofree.Trans where
```

Comment:
Same here - odd choice of module name?

Reply:
Whoops, missed that in review. I agree, the module name should be changed.